src/cml.js

```diff
@@ -208,7 +209,11 @@ class CML {
           );
           if (!triggerFile && watch) watcher.add(absolutePath);
           try {
-            node.url = await this.publish({ ...opts, path: absolutePath });
+            node.url = await this.publish({
```

Comment:
Should we have  a test for this?

Reply:
Would https://github.com/iterative/cml/pull/1166/commits/75bb629362c46765cd4d37e1a4cef817460229e2 be enough?

Reply:
won't we be uploading artifacts for every test run?

Reply:
*Cough,* yes, in negligible proportion when compared to the legacy `cml publish` tests. 😅 

https://github.com/iterative/cml/blob/dc6749c3fb79a5052cf3ebc3d77a9f4b940da7cd/bin/cml/asset/publish.test.js#L33-L109